triton-server/src/download/actions.ts

```diff
@@ -210,6 +210,21 @@ export async function createActions(db: Kysely<Database>) {
             .executeTakeFirstOrThrow()
     }
 
+    async function getLatestValidatedNotificationDate() {
+        return await db
+            .selectFrom("notification_dates")
+            .select("last_validated_notification_date")
+            .executeTakeFirstOrThrow()
+    }
+
+    async function updateLatestValidatedNotificationDate(date: string) {
+        return await db
+            .updateTable("notification_dates")
+            .set({ last_released_notification_date: date })
+            .returning("last_validated_notification_date")
+            .executeTakeFirstOrThrow()
+    }
+
```

Comment:
so you set `last_released_notification_date` but then you return `last_validated_notification_date`?

Reply:
my mistake :)